### PR TITLE
Tags molecule component added

### LINKED ISSUE #1105: Create a molecule component for tags
The component should be able to 
- [ ] List the available tags
- [ ] Create new tags
- [ ] Show the selected tags

@mayank111255 _a : Pick this up after the backend changes for neeto tags engine are done.
You can take help from @karthiknmenon and @amaldinesh7 on how to organize code and for review

SCR-20220420-vqg

SCR-20220420-vq4

### PR DESCRIPTION
**Pull request by mayanksharma4132**
closes #1105

### CODE REVIEW

**Comment by mayanksharma4132** [comment]
@karthiknmenon _a
Please check.

**Comment by karthiknmenon** [comment]
@praveen-murali-ind _a Please review the UI.

**Comment by praveen-murali-ind** [comment]
LGTM. @mayanksharma4132 _a

- [ ] Let’s allow users to choose a Tag style

//cc @karthiknmenon @amaldinesh7

**Comment by mayanksharma4132** [comment]
@praveen-murali-ind _a
Please check.

**Comment by praveen-murali-ind** [comment]
@mayanksharma4132 _a

**Comment by mayanksharma4132** [comment]
@karthiknmenon _a
Please check.

File: lib/molecules/Tags/index.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    ) : (
+      <Button
+        style="link"
+        label="New Tag"
+        icon={Plus}
+        onClick={() => setAddTag(true)}
+        {...buttonProps}
+      />)}
+  </div>;
+};
```
Comment (karthiknmenon) [inline lib/molecules/Tags/index.js:75]: Was the surrounding `()` removed by ESLint?

File: stories/Components/Tags.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+
+import Tags from '../../lib/molecules/Tags';
+
+export default {
```
Comment (karthiknmenon) [inline stories/Components/Tags.stories.jsx:5]: Let's keep this as `Molecules/Tags`.

**Review by karthiknmenon — APPROVED** [review]
@mayanksharma4132 _a Two minor comments. Please check. 

Please assign the PR to Amal for a final review.

File: lib/molecules/Tags/index.js (hunk 1 above)
Comment (mayanksharma4132) [reply lib/molecules/Tags/index.js]: No, It was not added, updated.

**Comment by mayanksharma4132** [comment]
@amaldinesh7 _a
Please check.

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@mayanksharma4132 _a Please check the comments. Add changelog for the change.

**Comment by mayanksharma4132** [comment]
@amaldinesh7 _a
Please check.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bigbinary/neetoui",
-  "version": "3.3.12",
```
Comment (amaldinesh7) [inline package.json:3]: Please remove this change, version update will be done by Publish Github action

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@mayanksharma4132 _A Please check the comment.

**Comment by mayanksharma4132** [comment]
@amaldinesh7 _a
Please check.

**Pull request merged**